flow/scripts/synth_hier_report.tcl

```diff
@@ -69,3 +75,4 @@ if { [info exist ::env(REPORTS_DIR)] && [file isfile $::env(REPORTS_DIR)/synth_h
   close $out_script_ptr
 }
 
+write_flatten_policy
```

Comment:
The naming feels a bit backwards throught.  The file contains the list of things _not_ to flatten.

Reply:
That to me is an implementation detail.

In a compiler we call it inlining policy regardless of how it is implemented. unflattening policy implies to me that the modules dont  exist, but have to be created... but they exist and we have a policy of which to flatten.

Reply:
You have promoted this to a result and not an implementation detail.  It was mark_hier_stop_modules.tcl; perhaps just keep_hierarchy.tcl